tests/settings.py

```diff
@@ -37,4 +34,5 @@
         },
     },
 ]
-FIREBASE_APP = initialize_app()
```

Comment:
Somehow initialization of `FIREBASE_APP` isn't required for test case scenarios.